website/docs/script/intro-to-script.mdx

```diff
@@ -24,9 +24,9 @@ A Script can be one of two types:
 - **Lock Script** - Used to control ownership and access to a Cell.
 - **Type Script** - Used to control how a Cell is used in a transaction.
 
-In most cases, Lock Script works the same with Type Script. The difference is that, only the Lock Script from the input Cells will be executed in the transaction, while the Type Script from both the input Cells and output Cells will be executed in the transaction.
+In most cases, Lock Script and Type Script function similarly. The only key takeaway is that, the Lock Scripts attached to the output Cells will not execute in a transaction.
```

Comment:
The only key difference is that, the Lock Scripts attached to the output Cells will not be executed in a transaction, whereas the Type Scripts are executed for both input and output Cells.

Reply:
Hi Linnnsss, if my change was too concise and straightforward, I suggest the following change. It's based on your comment and the original writing and lists all possible combinations.